src/geoms/constants.ts

```diff
@@ -6,4 +6,15 @@ export const colors = {
   copper: [0.9, 0.6, 0.2],
   fr4Green: [0x05 / 255, 0xa3 / 255, 0x2e / 255],
   fr4GreenSolderWithMask: [0x00 / 255, 0x98 / 255, 0x13 / 255],
+  fr1Copper: [0.8, 0.4, 0.2],
+  fr1CopperSolderWithMask: [0.9, 0.6, 0.2],
 } satisfies Record<string, RGB>
+export const boardMaterialColors: Record<string, RGB> = {
```

Comment:
When you use strong types like this, it ensures any future materials with throw a type error if they don't have a corresponding color (good)

Leveraging the type system is a huge advantage for maintaining code

```suggestion
export const boardMaterialColors: Record<Material, RGB> = {
```